lib/nacelle-pushowl-plugin.js

```diff
@@ -32,11 +32,40 @@ const cartToPushOwlObject = (cart) => {
   }
 }
 
-const handleProductView = (product) => {
+const handleProductView = (product, currentVariant) => {
   let productId = parseInt(decodeBase64Id(product.pimSyncSourceProductId), 10)
+
   window.pushowl.trigger('syncProductView', {
     productId
   })
+
+  if (!currentVariant.availableForSale) {
+    window.pushowl.trigger('showWidget', {
+      type: 'backInStock',
+      product: {
+        id: productId,
+        title: product.title
+      },
+      currentVariant: {
```

Comment:
@Rafi993 thinking more here, I think the API key should be just `variant`. `currentVariant` is our term which got propagated here  :) `current` makes sense for us, not for the developer who wants to subscribe to a product variant. wdyt?

Reply:
I used `variant` initially but changed to `currentVariant` to be more explicit. But thinking back now we can change it to `variant`

Reply:
Changed now.